.github/versions.yml

```diff
@@ -4,11 +4,11 @@ ruby/setup-ruby:
   :tag: v1.190.0
   :sha: a6e6f86333f0a2523ece813039b8b4be04560854
 actions/checkout:
-  :tag: v4.1.2
-  :sha: 9bb56186c3b09b4f86b1c65136769dd318469633
-nga333/variable-mapper:
+  :tag: v4.1.7
+  :sha: 692973e3d937129bcbf40652eb9f2f61becf3332
+kanga333/variable-mapper:
   :tag: v0.3.0
-  :sha: 3681b75f5c6c00162721168fb91ab74925eaebcb
+  :sha: c140b458cb69bd8c9c5eccd41e83aadc597c1352 # This commit is not on the main branch. It uses Node.js 20, which is needed to stop deprecation warnings.
```

Comment:
I think we should stick with commits that at least belong to the repo and not to a fork. A feature branch might be ok, but I think we should draw the line at forks.

Reply:
The obvious exception would be a `newrelic` owned fork...

Reply:
I see how a fork might be a little too saucy. I don't really know how to get newrelic to fork something lol! I'll revert this. Are we comfortable creating a separate backlog issue for this one and revisiting? We could even move this action into our `.github` folder like how I was thinking with [Create local technote-space/workflow-conclusion-action gha](https://github.com/newrelic/newrelic-ruby-agent/issues/2790)

Reply:
> Are we comfortable

I'm not sure of what the deadline is and what happens when after the deadline. If the deadline is far away, we're comfortable. If the deadline being reached simply causes the action to be forced to run on a Node v20 runtime, we're comfortable. If the deadline is soon and/or it involves the action failing instead of simply running under v20, then we'll have to evaluate urgency.

I see at least these 5 ways forward (not listed in any intentional order) - all of which I am happy to respect the group consensus on:

- Override me and go with this PR as-is. Perhaps throw me a bone and at least put a "TODO: " prefix in the .yml file's comment line and create a backlogged issue for the TODO.
- Create the separate backlog issue and revisit as you mentioned.
- Point to the fork instead of the original repo. It's really the mixing of 2 repos that I have an issue with. Unless we have a particularly strict process for allowlisting new actions repos, we could just point to the fork. Unfortunately the rake task isn't smart enough to handle switching repos and the workflow lines would need to be manually changed.
- Fork the action under your name and point there until we can get the repo moved under `newrelic/`
- Fork and vendor the action under `.github` as you mentioned

Reply:
Looks like Jan 6, 2025 is end of support. I'll remove this and add these comments to the ticket.

Reply:
Linked this convo to the [kanga333 ticket](https://github.com/orgs/newrelic/projects/84/views/1?pane=issue&itemId=71162310).